package.json

```diff
@@ -214,7 +215,7 @@
   "size-limit": [
     {
       "path": "commonjs/index.js",
-      "limit": "241 KB"
+      "limit": "260 KB"
```

Comment:
Dang -- how come the bundle size bump?

Reply:
I think that `emotion` is actually smaller than glamor, but then adding `emotion-server` for SSR support bloats the commonjs bundle. Fortunately the esm bundle shouldn't be impacted since that will get tree-shaken.